babel.config.js

```diff
@@ -25,6 +25,7 @@ const PLUGINS = [
   '@babel/plugin-transform-modules-commonjs',
   '@babel/plugin-proposal-class-properties',
   '@babel/plugin-proposal-export-namespace-from',
+  '@babel/plugin-proposal-optional-chaining',
```

Comment:
Guessing this may be needed in `package.json`?

Reply:
it is a dependency of `@babel/preset-env`, but worth calling it out i guess